### DISCUSSION TITLE
Is xtls-rprx-vision include xtls-rprx-splice Linux performance boost?

### DISCUSSION THREAD

**Discussion by GektorUA** [XTLS/Xray-core#1604] [Q&A]
I see that in wiki xtls-rprx-splice is deprecated now, is xtls-rprx-vision include Linux splice boost performance for non encrypted by TLS traffic?

**Comment by RPRX** [comment]
It does. Flow `xtls-rprx-vision` will use Linux Splice automatically. No further configuration is required.
